[VA-5079] Update media xdm fields

lidiaist opened the pull request:
Issue https://github.com/adobe/xdm/issues/1550

lidiaist commented:
@mjasrotia Please review. We made a few updates to our experimental media fieldgroups after starting our backend implementation. Thank you!

prabhum2 commented on components/datatypes/qoedatadetails.schema.json:
```diff
@@ -15,66 +15,86 @@
-          "type": "number",
-          "description": "The number of streams in which an error event occurred."
+          "type": "boolean",
+          "description": "The number of streams in which an error event occurred (i.e., trackError was called during the playback session, and a type=error heartbeat call was generated)."
```
If this is boolean , Can you update description reflecting latest changes

lidiaist replied:
@prabhum2 Done, thank you!

prabhum2 commented:
@lidiaist let me know when this is ready to be reviewed and merged . I see more changes after we approved

lidiaist commented:
@prabhum2 Done. I made the last updates. If I need any further updates in the future, I will create a new PR, so if you could please merge this PR it would be very helpful. Thank you!

lidiaist commented:
In the last commit, I added the existing endUserIDs and implementationDetails datatypes to our mediaCollection fieldgroup and the implementationDetails datatype to our mediaReporting fieldgroup because this makes it easier for our clients to know which fields to complete for each group of either collection or reporting.  (Currently I added these 2 in the AEP schema as individual fieldgroups separated from mediaCollection and mediaReporting but using them as datatypes inside our fieldgroups makes our job easier). Thank you!
